syndeo-template/README.md

```diff
@@ -14,7 +14,7 @@ The template works by relying on Beam transforms that implement the `SchemaTrans
 template's classpath.
 
 The template receives a *pipeline spec*, which is a specification defining the pipeline's source, sink, and intermediate
-transforms. Then, the template will take the pipeline spec, and
+transforms. Then, the template will take the pipeline spec, and TODO(nickllx): fill this part.
```

Comment:
possible to fix the TODO before we submit this change?

Reply:
@pabloem hey Pablo would you mind filling this TODO here? I'm not a 100% sure about the entire schema transform workflow

Reply:
can we submit this PR. maybe remove this for now?

Reply:
Done